decidim-proposals/spec/events/decidim/proposals/proposal_endorsed_event_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::Proposals::ProposalEndorsedEvent do
+  subject do
```

Comment:
It also suggest to use the shared examples (simple event) for this.

Reply:
Done